tyk-docs/data/menu.yaml

```
@@ -2001,7 +2001,7 @@ menu:
               path: /product-stack/tyk-gateway/advanced-configurations/distributed-tracing/open-tracing/zipkin
               category: Page
               show: True
-        - title: "Plugins"
+        - title: "Developing plugins"
```

```suggestion
        - title: "Custom plugins"
```

I would go with "custom plugins" here because that's consistently the term used to describe them and so is more obvious to the reader to find.